README.md

```diff
@@ -21,10 +21,10 @@ SimpleLibrary
 Now to generate cpp bindings
 ```
 $ cd godot-cpp
-$ scons godotbinpath="../godot_fork/bin/godot_binary" p=linux
+$ scons godotbinpath="../godot_fork/bin/godot_binary" p=linux headers=../godot_headers
```

Comment:
Actually a recent change made it so that it should only be `scons p=linux` and then you're done.